### PR TITLE
Add ng support

### PR DESCRIPTION
**Pull request by frontegg-david**

### CODE REVIEW

**Review by aviadmizrachi — CHANGES_REQUESTED** [review]

File: packages/auth/src/components/authPageWrapper.tsx [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
+import { HeaderProps } from '../interfaces';
 
-export const authPageWrapper = <P extends {}>(
-  Component: ComponentType<P>
-): ComponentType<P & { header?: ReactNode }> => (props: P & { header?: ReactNode }) => {
-  const { header = <img src='http://acmelogos.com/images/logo-1.svg' alt='logo' /> } = props;
+export const authPageWrapper = <P extends {}>(Component: ComponentType<P>): ComponentType<P & HeaderProps> => (
+  props: P & HeaderProps
+) => {
+  const header = props.header ?? <img src={props.headerImg ?? '//acmelogos.com/images/logo-1.svg'} alt='logo' />;
```
[the remark this answers is not in the document]
Comment (frontegg-david) [reply packages/auth/src/components/authPageWrapper.tsx:8]: what about localhost with https?

[the remark this answers is not in the document]
Comment (frontegg-david) [reply packages/auth/src/components/authPageWrapper.tsx:8]: Tested it on localhost it's working

**Review by aviadmizrachi — APPROVED** [review]

**Pull request merged**